demo-projects/meetup/site/lib/authetication.js

```diff
@@ -131,15 +136,25 @@ const signout = async () => {
   }
 };
 
-const checkSession = async () => {
+const checkSession = async apolloClient => {
   try {
-    const res = await fetch(`${apiEndpoint}/session`, {
-      method: 'GET',
-      credentials: 'same-origin',
-    }).then(r => r.json());
+    const {
+      data: { authenticatedUser },
+    } = await apolloClient.query({
+      query: gql`
+        query {
+          authenticatedUser {
+            id
+            name
+            isAdmin
```

Comment:
Why these fields? They are the same as what was previously set in:

https://github.com/keystonejs/keystone-5/pull/1174/files#diff-2d3982e1bacb1e9fcab76daf0372ee65L27